src/build.sh

```diff
@@ -56,7 +56,7 @@ cd "$SRC_DIR"
 
 if [ -f /root/userscripts/begin.sh ]; then
   echo ">> [$(date)] Running begin.sh"
-  /root/userscripts/begin.sh || echo ">> [$(date)] Warning: begin.sh failed!"
+  /root/userscripts/begin.sh
```

Comment:
Why remove the warning here?

Reply:
The OR operator suppresses the error and allows the script to continue. Without it the script will exit immediately due to `set -e`.

If keeping the warning is important, an alternative implementation would be:

```sh
/root/userscripts/begin.sh || { echo ">> [$(date)] Error: begin.sh failed!"; exit 1; }
```

Reply:
> /root/userscripts/begin.sh || { echo ">> [$(date)] Error: begin.sh failed!"; exit 1; }

That sounds like a better solution: otherwise the build continues after `begin.sh` fails, which is the problem you want to solve

Reply:
Thanks for the change